### ISSUE TITLE
Feature Request: Default volume on startup

### ISSUE THREAD

**Issue by yash1610** [u8sand/Baka-MPlayer#217]
Currently whenever the program starts the volume is set at 100, if you open a file using your file-manager sure enough the program starts at 100% volume deafening me and waking up everyone in the house. I couldn't find an obvious way to change the default volume on startup. Tried putting 
`'volume': 30,` in `~/.config/bakamplayer.ini` but that doesn't seem to work, it still starts at full volume.

**Comment by ErikDavison** [comment]
Getting the same problem. The .ini file seems to be getting read in and updated fine so I don't think its that. I've done a little debugging so far, and I've found that mpv's volume gets updated twice at startup: 1st, to the correct value in .ini, and then again, to 100. I'll try and pin down where this 2nd event is coming from.

**Comment by ErikDavison** [comment]
After a little more digging I found that the 2nd event was coming after mpv was initialized. I found that
mpvhandler's volume was being loaded by the .ini, however this never got reflected in the actual mpv instance. Thus after initialization mpv was loading a default value of 100.

The patch I've submitted ensures mpvhandler's volume is registered with mpv when there is no playback (such as before initialization).
